BrownianMotion/StochasticIntegral/ClassD.lean

```diff
@@ -95,9 +104,15 @@ section LinearOrder
 
 variable [LinearOrder ι] {𝓕 : Filtration ι mΩ}
 
-lemma isStable_hasLocallyIntegrableSup [OrderBot ι] [TopologicalSpace ι] [OrderTopology ι] :
-    IsStable 𝓕 (HasLocallyIntegrableSup (E := E) · 𝓕 P) := by
-  sorry
+lemma isStable_hasIntegrableSup [OrderBot ι] [TopologicalSpace ι]
+    [OrderTopology ι] [MeasurableSpace ι] :
+    IsStable 𝓕 (HasIntegrableSup (E := E) · P) := by
+      sorry
+
```

Comment:
Can you add a lemma here (using `locally_and`) to say that one has `HasLocallyIntegrableSup` if and only if one have (non-local I think) measurability and local integrability

Reply:
My impression is that you'd never want to talk about (integrable sup) without the measurability condition, as it won't be stable.